### ISSUE TITLE
can mind take inputs that vary from 0-1? (ex: 0.2, 0.3, 0.5, etc)

### ISSUE THREAD

**Issue by vynci** [stevenmiller888/mind#13]
can mind take inputs that vary from 0-1? (ex: 0.2, 0.3, 0.5, etc)

like this example from brain.js:

var net = new brain.NeuralNetwork();

net.train([{input: { r: 0.03, g: 0.7, b: 0.5 }, output: { black: 1 }},
           {input: { r: 0.16, g: 0.09, b: 0.2 }, output: { white: 1 }},
           {input: { r: 0.5, g: 0.5, b: 1.0 }, output: { white: 1 }}]);

var output = net.run({ r: 1, g: 0.4, b: 0 });  // { white: 0.99, black: 0.002 }

...i really like mind.js because of it's simplicity compared with brain.js...but i just couldn't get a proper output..thanks!

**Comment by stevenmiller888** [comment]
Thanks! I'm glad you like it. With regards to your question, it depends on how you configure the network. If you use the sigmoid activation function, then the network can take inputs from 0 to 1. If you use the hyperbolic activation function, then the network can take inputs from -1 to 1. 

In terms of getting a proper output, you'll have to jiggle with the network hyperparameters a bit. Try both activation function types, try changing the number of learning iterations, and try changing the learning rate. 

Hope that helps!

**Comment by vynci** [comment]
thanks for the quick reply @stevenmiller888! I tried lowering the learningRate to 0.3, and increased the default 10000 learning iterations..i think i'm getting more constant and better results already

By the way i'm trying out the famous Iris data set from UCI. I used this formula for normalization, before training them: 

```
        [x(i)-min(x)]
x(i) =  ------------------- --> values for each value in that vector variable (column)
        [max(x)-min(x)]
```

i kinda did this because the range of the numeric (continuous or integer) variable is outside the scope of sigmoid output, e.g., e^x which is 0 to 1 OR tanh which is -1 to +1...well i hope i'm on the right path hehe

Your source code is really very easy to understand, code readability is brilliant. I learned a lot from your tutorials in your blog as well. Thanks Again!
